app/data_models/progress_store.py

```diff
@@ -84,7 +94,7 @@ def is_section_complete(
 
     def section_keys(
         self, statuses: Iterable[str] = None, section_ids: Iterable[str] = None
-    ):
+    ) -> list:
```

Comment:
As per type hinting guide, if we can, define the types for all collections.

Reply:
Defined, thanks.